frontend/src/api/tus.js

```diff
@@ -48,3 +46,40 @@ export async function upload(url, content = "", overwrite = false, onupload) {
     });
   });
 }
+
+function computeRetryDelays(tusSettings) {
+  if (!tusSettings.retryCount || tusSettings.retryCount < 1) {
+    // Disable retries altogether
+    return null;
+  }
+  return Array.apply(null, { length: tusSettings.retryCount }).map(
+    Number.call,
+    (idx) => (idx + 1) * tusSettings.retryBaseDelay * tusSettings.retryBackoff
+  );
```

Comment:
This section should be simplified to reduce confusion. Also a comment to explain what its doing would be helpful.
```suggestion
  return Array.apply(null, { length: tusSettings.retryCount }).map(
    (_, idx) => (idx + 1) * tusSettings.retryBaseDelay * tusSettings.retryBackoff
  );
```